app/activity/create/components/form.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useCallback, useState } from 'react'
+import { useDropzone } from 'react-dropzone'
+import { useForm } from 'react-hook-form'
+import {
+  Box,
+  Container,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  Input,
+  Button,
+  Textarea,
+  Image,
+  SimpleGrid,
+  Text,
+  Flex
+} from '@chakra-ui/react'
+
+import { PrimaryButton } from '@/components/button'
+import { createActivitySchema, createActivityResolver } from './schema'
+
+export const Form = () => {
```

Comment:
I think it's better to say `FormActivity` which is more clearly and easier to search!

Reply:
I changed the name to FormActivity!